kuberay-ray-operator.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: kuberay-ray-operator
+  version: 1.0.0
+  epoch: 0
+  description: A toolkit to run Ray applications on Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 1add258fcfd5c83319428355c0af4dde60ea9047
+      repository: https://github.com/ray-project/kuberay.git
+      tag: v${{package.version}}
+
+  - runs: |
+      # https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md
+      export BUILD_TIME=$(date "+%F %T")
+      export COMMIT_SHA1=$(git rev-parse HEAD)
+      cd ray-operator
+      rm -rf bin
+      go fmt ./...
+      go vet ./...
+      go build -ldflags "-X 'main._buildTime_=$BUILD_TIME' -X 'main._commitId_=$COMMIT_SHA1'" -o bin/manager main.go
+
+  - uses: strip
+
```

Comment:
we can add simple `test:` pipeline here and run something like `/usr/bin/manager --help` or `/usr/bin/manager version`.

Reply:
added testing and successful results locally